contracts/cw1-subkeys/src/multitest/tests.rs

```diff
@@ -124,7 +127,7 @@ mod allowance {
             .call(owner)
             .unwrap();
 
-        let height = app.block_info().height;
+        let height = (*app).borrow().block_info().height;
```

Comment:
Better to implement `block_info()` directly like before or use `.app()`; this Deref looks bad. It is for all the instances, would comment once.

Reply:
Requires https://github.com/CosmWasm/cw-multi-test/pull/43